src/pages/Form/SingleForm.tsx

```diff
@@ -1,52 +1,62 @@
 import React, { useState } from "react";
-import { useGetFormQuery } from "../../features/user/apiSlice";
+import { useGetFormQuery } from "../../features/user/backendApi";
 import Loader from "../../components/ui/Loader";
-import { useNavigate, useParams } from "react-router-dom";
+import { useParams } from "react-router-dom";
 import EditForm from "../../components/ui/EditForm";
-import Back from "../../assets/BackIcon";
 import QuestionCard from "../../components/ui/QuestionCard";
-import { getJWT } from "../../utils/helper";
+import { Cookie, Question } from "../../utils/types";
+import { useCookies } from "react-cookie";
 
 const SingleForm = () => {
-  const navigate = useNavigate();
-  const id = useParams().id || "";
-  const jwt:string = getJWT()
-  const { data, isFetching } = useGetFormQuery({ id, jwt });
-  const name = data?.name;
-  const description = data?.description || "";
-  const questions = data?.questionIds || [];
-  const [activeQuestion, setActiveQuestion] = useState("");
+  const [cookies] = useCookies([Cookie.jwt]);
+  const { id } = useParams<{ id: string }>();
+  const { data, isFetching, error } = useGetFormQuery({
+    id: id || "",
+    jwt: cookies.jwt,
+  });
+
+  const [activeQuestion, setActiveQuestion] = useState<string>("");
+
+  if (isFetching) {
+    return (
+      <div className="h-[50vh] flex items-center justify-center">
```

Comment:
Its bad practices to use `[...]` because it increases the code size. Can you use standard tailwind classes instead?